src/maintainer/infrastructure.rst

```diff
@@ -25,6 +25,10 @@ Smithy contains maintenance code for conda-forge, which is used by the ``conda-s
 
 ``conda-smithy`` also contains the command line tool that you should use if you rerender manually from the command line (see :ref:`dev_update_rerender`).
 
+Smithy can be used for more than just CF; it can also be used to set up self-hosted agents for non-CF infrastructures.
```

Comment:
I don't think we refer to "CF" in this file, so let's avoid it:
```suggestion
Smithy can be used beyond Conda-Forge's purposes. For example, it can be used to set up self-hosted Azure agents for non-Conda-Forge infrastructures.
```